chapter_recurrent-neural-networks/encoder-decoder.md

```diff
@@ -65,8 +64,9 @@ class EncoderDecoder(nn.Block):
 ## Summary
 
 * An encoder-decoder architecture is a neural network design pattern mainly in natural language processing.
-* An encoder is a network (FC, CNN, RNN, etc) that takes the input, and output a feature map, a vector or a tensor.
-* An decoder is a network (usually the same network structure as encoder) that takes the feature vector from the encoder, and gives the best closest match to the actual input or intended output.
+* An encoder is a neural network (FC, CNN, RNN, etc.) that takes the input, and output a feature map.
+* An decoder is a network usually the same network structure as encoder. 
```

Comment:
An decoder is a network usually the same network structure as encoder. 

this is not a sentence

Reply:
ok